package.json

```diff
@@ -305,7 +305,7 @@
     "uswds": "1.6.10",
     "vanilla-lazyload": "^16.1.0",
     "vets-json-schema": "https://github.com/department-of-veterans-affairs/vets-json-schema.git#3a0946727d9013ff2771af37bfca0c4c0ff0a1d5",
-    "web-components": "https://github.com/department-of-veterans-affairs/component-library/releases/download/wc-v0.11.1/web-components-v0.11.1.tgz",
+    "web-components": "https://github.com/department-of-veterans-affairs/component-library#6f34f06d5bbba2a216dcfbca535169ccaec5d4d4",
```

Comment:
Release associated with tar https://github.com/department-of-veterans-affairs/component-library/releases/tag/wc-v0.11.1